### DISCUSSION TITLE
Creating a WiX4 msi with no project file for each msi

### DISCUSSION THREAD

**Discussion by DSmitj** [wixtoolset/issues#7674] [Questions] [answered]
I haven't been able to find anything on the subject of creating WiX4 installs without a project file.

I've got several installs in WiX3 that I want to convert to Wix4.  The current installs don't have a project file associated with them.  Is this possible in WiX4?  Am I going to need to create a project file for each install?  Am I looking at a total rewrite incorporating a wixproj file for each install?

Thanks,
D

**Comment by robmen** [comment]
Quick question: Have you read the "Get started with WiX" page?

**Reply by DSmitj** [reply]
I've scanned through that and a lot of the other documentation like going from WiX3 to WiX4.

I noted that there was a lot of inclusion of the wixproj file for a lot of things related to WiX4.  The tools have been streamlined into wix.exe with the exception of the heattransform.

WiX3 you could create msies without a wixproj file for each msi.

I figured that I'd ask it directly of you folks so I could get an idea of the scope of work I had ahead to move to WiX4.  I'm not trying to waste anyones' time.  Just wanted to get a definitive answer.

**Reply by robmen** [reply]
Yep. That's it.

I'm curious why the documentation wasn't enough.

**Reply by DSmitj** [reply, marked as answer]
The ole colloquial adage that there is more than one way to skin a cat (not that I'd ever try).  I'd figured I'd ask if your team were doing use cases beyond just the wixproj.

Thanks for your consideration,
D

**Reply by robmen** [reply]
You might watch the [Deployment Dojo shows](https://bit.ly/github-discussions-dojo-playlist). You'll see I use both methods and the strengths of each.
